holidays/countries/andorra.py

```diff
@@ -19,10 +21,60 @@ class Andorra(HolidayBase, ChristianHolidays, InternationalHolidays):
 
     References:
         * <https://en.wikipedia.org/wiki/Public_holidays_in_Andorra>
-        * <https://www.holsdb.com/public-holidays/ad>
+        * [Andorra's Constitution](https://web.archive.org/web/20250506145838/https://www.bopa.ad/Legislacio/Detall?doc=7586)
+        * [2025](https://web.archive.org/web/20250506142149/https://www.bopa.ad/Documents/Detall?doc=GD_2024_10_24_14_36_16)
```

Comment:
From https://seu.consellgeneral.ad/calendariPublic/show there's also `HALFDAY` categories here, maybe a common `, a partir de les 13h` (from 1 pm) suffix for all of these entries wouldn't hurt

> L'horari d'atenció al públic serà de les 8h a les 13h. els dies que sigui festiu la tarda:
17 d'abril, Dijous Sant (Holy Thursday)
24 de desembre, vigília de Nadal (Christmas Eve)
31 de desembre, vigília de Cap d'Any (New Year's Eve)
5 de gener del 2026, vigília de Reis (Epiphany's Eve)